apps/framework-cli/Cargo.toml

```diff
@@ -36,6 +36,7 @@ uuid = { version = "1.6", features = ["v4"] }
 serde_json = "1.0.108"
 serde_urlencoded = "0.7"
 base64 = "0.22.0"
+crypto-hash = "0.3.4"
```

Comment:
It looks like this package might not be maintained, last release was 3 years ago.

I would advise this one instead: https://crates.io/crates/sha2